ServiceInnovationLab/FamilyServiceDirectorySearchinator
# Specify node version 9.8.0

# What has changed and why?

Each dev is running a different version of node. This is a bad idea. This file will tell nvm which version to run for this project.

# How to test this change

Check out branch, delete node_modules dir, `npm install`. run the app and test it still works
